### PR TITLE
feat: add optimism sepolia testnet [APE-1488]

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did

requires: https://github.com/ApeWorX/ape-optimism/pull/19

### Checklist

- [x] Passes all linting checks (pre-commit and CI jobs)
- [x] New test cases have been added and are passing
- [x] Documentation has been updated
- [x] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

File: ape_etherscan/utils.py [hunk 1]
```diff
@@ -11,42 +11,43 @@
...
+    "fantom": [
+        "opera",
+        "testnet",
     ],
-    "polygon-zkevm": [
+    "gnosis": ["mainnet"],
+    "optimism": [
         "mainnet",
         "goerli",
+        "sepolia",
```
Comment (antazoey) [inline ape_etherscan/utils.py:43]: only addition here, rest of diff is sorting

**Review by antazoey — APPROVED** [review]

**Review by antazoey — DISMISSED** [review]

**Comment by fubuloubu** [comment]
What's going on with that new tokens file?

**Comment by antazoey** [comment]
> What's going on with that new tokens file?

no idea - deleted though

**Comment by antazoey** [comment]
tests were broken in a few ways - also was an issue with version types.
all should be golden now.

**Review by antazoey — APPROVED** [review]

**Pull request merged**